environment-api/src/main/java/com/sequenceiq/environment/api/v1/environment/validator/EnvironmentNetwork.java

```diff
@@ -0,0 +1,22 @@
+package com.sequenceiq.environment.api.v1.environment.validator;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import javax.validation.Constraint;
+import javax.validation.Payload;
+
+@Documented
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.RUNTIME)
+@Constraint(validatedBy = EnvironmentNetworkValidator.class)
+public @interface EnvironmentNetwork {
```

Comment:
I think the name is not meaningful enough. something like `ValidEnvironmentNetwork` is indicating more what is the purpose of this class

Reply:
I have removed it